src/main/java/pl/cleankod/exchange/core/usecase/AccountServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+
+package pl.cleankod.exchange.core.usecase;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import pl.cleankod.exchange.core.domain.Account;
+import pl.cleankod.exchange.core.domain.Money;
+import pl.cleankod.exchange.core.gateway.AccountRepository;
+import pl.cleankod.exchange.core.gateway.AccountService;
+import pl.cleankod.exchange.core.gateway.CurrencyConversionService;
+import pl.cleankod.exchange.entrypoint.model.AccountNotFoundException;
```

Comment:
Should the `core` module depend on the `entrypoint` module?

Reply:
no